gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
```

Comment:
Changing the URL here is not sufficient to really update gradle. 
You need to run `./gradlew wrapper` as described here: https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper

Reply:
EDIT: You should

Reply:
Thanks.  Actually ran the command and pushed: 2912e7a

I also created a quick 'how-to-upgrade' gradle doc and pushed that: 31150eb

Please have a look if you would in case I left anything out of the steps.